scripts/rollup/validate/eslintrc.umd.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     define: true,
     require: true,
     global: true,
+    globalThis: true,
```

Comment:
Why? This isn't a trivial change. It's Stage 3 which means it still can change. But React using it in a release would effectively seal its semantics.

Can you clarify how `globalThis` is used in the output, and why it's safe to do it now? cc @ljharb

Reply:
Stage 3 or not, it's shipped in every browser, so its semantics are already sealed.

Reply:
Thanks for clarifying.

Reply:
I added `globalThis` because of this
https://circleci.com/gh/facebook/react/38972?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Could you show react-art.development.js?

Reply:
The usage of globalThis is here
https://github.com/kunukn/reactjs-rollup-upgrade/blob/rollup.1.19.4/build/node_modules/react-art/umd/react-art.development.js#L966

This looks fine to me.